src/mem/cache/tags/lru.cc

```diff
@@ -176,7 +177,35 @@ LRU::findVictim(Addr addr)
 {
     unsigned set = extractSet(addr);
     // grab a replacement candidate
-    BlkType *blk = sets[set].blks[assoc-1];
+    // BlkType *blk = sets[set].blks[assoc-1];
+
+    // The replacement will be found from left to right. 
+    // The first blk with RRPV of assoc - 1 will be the replaced blk
```

Comment:
Why is this assoc - 1? I thought the limit was always 3. Or was that just for a two bit SRRIP?